.github/workflows/plugindocs-update.yml

```diff
@@ -0,0 +1,63 @@
+name: Update Plugin Docs
+on:
+  schedule:
+    - cron: '0 9 * * *'
+    workflow_dispatch:
```

Comment:
What does this line do?

Reply:
That apparently allows manual triggering of the action